### PR TITLE
cleaning project

### PR DESCRIPTION
**Pull request by Lowentwickler**

### CODE REVIEW

File: templates/delete.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
...ule:evenodd; clip-rule:evenodd' viewBox='0 0 14.61 26.97' xmlns:xlink='http://www.w3.org/1999/xlink'%3E %3Cdefs%3E %3Cstyle type='text/css'%3E %3C!%5BCDATA%5B .fil0 %7Bfill:%23EF7F1A%7D .fil1 %7Bfill:white%7D %5D%5D%3E %3C/style%3E %3C/defs%3E %3Cg id='Layer_x0020_1'%3E %3Cmetadata id='CorelCorpID_0Corel-Layer'/%3E %3Cpath id='temperature.svg' class='fil0' d='M11.24 19.67c0,2.17 -1.76,3.93 -3.93,3.93 -2.17,0 -3.93,-1.76 -3.93,-3.93 0,-1.29 0.68,-2.35 1.57,-3.13 1.13,-0.99 1.81,-1.95 1.81,-3.62l0 -0.56 1.13 0 0 0.56c0,1.67 0.66,2.63 1.81,3.62 0.89,0.78 1.57,1.84 1.57,3.13l-0.02 0z'/%3E %3Cpath id='temperature.svg_0' class='fil1' d='M11.24 3.93l0 8.45c0,0.66 0.3,1.29 0.8,1.72 1.58,1.34 2.57,3.34 2.57,5.57 0,4.03 -3.27,7.3 -7.3,7.3 -4.03,0 -7.3,-3.27 -7.3,-7.3 0,-2.23 1.01,-4.23 2.57,-5.57 0.5,-0.43 0.8,-1.06 0.8,-1.72l0 -8.45c0,-2.17 1.76,-3.93 3.93,-3.93 2.17,0 3.93,1.76 3.93,3.93l0 0zm-5.62 0l0 8.45c0,1.76 -0.57,2.57 -1.58,3.43 -1.13,0.97 -1.79,2.37 -1.79,3.84 0,2.78 2.28,5.06 5.06,5.06 2.78,0 5.06,-2.28 5.06,-5.06 0,-1.48 -0.64,-2.89 -1.79,-3.84 -1.01,-0.85 -1.58,-1.67 -1.58,-3.43l0 -8.45c0,-0.92 -0.77,-1.69 -1.69,-1.69 -0.92,0 -1.69,0.77 -1.69,1.69l0 0z'/%3E %3C/g%3E %3C/svg%3E">
```
Comment (filipkotoucek) [inline templates/delete.html:44]: Why do we need to import SVG directly in HTML? I would rather add them as asset.

File: scripts/extractLang.js [hunk 2]
```diff
@@ -1,86 +0,0 @@
-// This file is part of Prusa-Connect-Local
```
Comment (filipkotoucek) [inline scripts/extractLang.js:1]: How the translations will be now updated?

File: templates/img/calendar.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16.284" height="14.929" viewBox="0 0 16.284 14.929"><defs><style>.a{fill:#f8651b;}.a,.b{fill-rule:evenodd;}.b{fill:#fff;}</style></defs><path class="a" d="M8.261,11.09H5.55v2.716H8.261Z" transform="translate(-2.834 -5.664)"/><path class="a" d="M16.581,19.41H13.87v2.716h2.711Z" transform="translate(-7.083 -9.913)"/><path class="b" d="M16.284,0V14.929H0V0ZM14.929,4.071H1.36v9.5H14.929Z"/><path class="a" d="M8.261,19.41H5.55v2.716H8.261Z" transform="translate(-2.834 -9.913)"/><path class="a" d="M24.9,11.09H22.19v2.716H24.9Z" transform="translate(-11.332 -5.664)"/><path class="a" d="M16.581,11.09H13.87v2.716h2.711Z" transform="translate(-7.083 -5.664)"/><path class="a" d="M24.9,22.126H22.19V19.41H24.9Z" transform="translate(-11.332 -9.913)"/></svg>
```
Comment (filipkotoucek) [inline templates/img/calendar.svg:1]: Already in assets. If you need the same file on multiple locations could you use links?

File: templates/delete.html (hunk 1 above)
Comment (brunoOnTheWorld) [reply templates/delete.html:44]: Please, ignore the templates/*. 
I should be renamed to draft, it's not the final code.

File: scripts/extractLang.js (hunk 2 above)
Comment (brunoOnTheWorld) [reply scripts/extractLang.js:1]: This extract from touch-ui, it's good to create a script for phrase.

File: src/index.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import "./styles.css";
 import { navigate } from "./router.js";
 import printer from "./printer";
 
-const UPDATE_INTERVAL = 5000;
```
Comment (brunoOnTheWorld) [inline src/index.js]: I think `const UPDATE_INTERVAL = 5000;` is better solution.

**Review by brunoOnTheWorld — APPROVED** [review]

**Pull request merged**